lms/static/sass/shared/_header.scss

```diff
@@ -627,8 +627,8 @@ header.global-new {
       a {
         display:block;
         padding: 3px 10px;
-        font-size: 18px;
-        line-height: 24px;
+        font-size: 14px;
+        line-height: 1.5;
```

Comment:
What are the units, or is this a multiplier? Will this pose issues for open source users?